interface-definitions/include/firewall/nat-balance.xml.i

```diff
@@ -0,0 +1,28 @@
+<!-- include start from firewall/nat-balance.xml.i -->
+<tagNode name="member">
```

Comment:
What does `member` stand for as the help states "translated" - maybe there is a better node name available?

Reply:
Ok, I'll change and go for second option:
```
set nat destination rule 10 load-balance hash 'source-address'
set nat destination rule 10 load-balance hash 'source-port'
set nat destination rule 10 load-balance backend 198.51.100.11 weight '25'
set nat destination rule 10 load-balance backend 198.51.100.21 weight '25'
set nat destination rule 10 load-balance backend 198.51.100.31 weight '50'
```